### PR TITLE
Release step 1: Update changelog file

### PR DESCRIPTION
**Pull request by tamiko**

### CODE REVIEW

File: doc/news/9.1.1-vs-9.2.0.h [hunk 1]
```diff
@@ -0,0 +1,2579 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 - 2020 by the deal.II authors
```
Comment (kronbichler) [inline doc/news/9.1.1-vs-9.2.0.h:3]: While here, I realize that we might want to change this to 2019 - 2020? Or how do we want this to be?

Comment (tamiko) [reply doc/news/9.1.1-vs-9.2.0.h]: fixed.

File: doc/news/9.1.1-vs-9.2.0.h [hunk 2]
```diff
@@ -0,0 +1,2579 @@
...
+  New: A second template parameter has been added to VectorizedArray. This template
+  parameter optionally controls the number of vectorization lanes to be used, i.e.,
+  which instruction set architecture extension to be used, at the time the application
+  is built.
+  <br>
+  (Peter Munch, 2019/07/04)
+ </li>
+
+ <li>
+  Improved: All member funtions and free functions related to Tensor
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:2423]:
```suggestion
  Improved: All member functions and free functions related to Tensor
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 3]
```diff
@@ -0,0 +1,2579 @@
...
+  Deprecated: SparsityTools::distribute_sparsity_pattern() and
+  SparsityTools::gather_sparsity_pattern() taking the locally owned index sets
+  of all processes as second arguments have been deprecated. Use the respective
+  functions only providing the locally owned dofs of the calling process.
+  <br>
+  (Martin Kronbichler, 2020/03/22)
+ </li>
+
+ <li>
+  Chaned: VectorizedArray::n_array_elements has been deprecated.
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:281]:
```suggestion
  Changed: VectorizedArray::n_array_elements has been deprecated.
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 4]
```diff
@@ -0,0 +1,2579 @@
...
+  colorbar and the caption. This is achieved by changing the defaults in
+  the constructor of the GridOutFlags::Svg structure. This likely comes
+  closer to what most users want to see from this function -- namely,
+  just the mesh for inclusion into text.
+  <br>
+  (Wolfgang Bangerth, 2019/08/05)
+ </li>
+
+ <li>
+  Removed: The deprecated PreconditionChebyshev member variables nonzero_strarting
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:526]:
```suggestion
  Removed: The deprecated PreconditionChebyshev member variables nonzero_starting
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 5]
```diff
@@ -0,0 +1,2579 @@
...
+
+ <li>
+  New: The step-47 tutorial program demonstrates the solution of the
+  biharmonic equation, a fourth-order differential equation.
+  <br>
+  (Natasha Sharma, Guido Kanschat, Timo Heister, Wolfgang Bangerth, Zhuoran Wang, 2020/01/15)
+ </li>
+
+ <li>
+  New: The SymbolicFunction<dim> class allows one to leaverage the SymEngine library to generate
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:696]:
```suggestion
  New: The SymbolicFunction<dim> class allows one to leverage the SymEngine library to generate
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 6]
```diff
@@ -0,0 +1,2579 @@
...
+ <li>
+  New: Add a flag to overlap MPI communication and computation when using CUDA
+  matrix-free and CUDA-aware MPI.
+  <br>
+  (Peter Munch and Bruno Turcksin, 2020/05/06)
+ </li>
+
+ <li>
+  Changed: The order by which MatrixFree::cell_loop() and MatrixFree::loop()
+  pass through cells has been changed to preferrably group cells with the same
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:828]:
```suggestion
  pass through cells has been changed to preferably group cells with the same
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 7]
```diff
@@ -0,0 +1,2579 @@
...
+  GridGenerator::hyper_ball().
+  <br> (Martin Kronbichler, 2020/04/02)
+ </li>
+
+ <li>
+  Changed: The vertex positions of GridGenerator::quarter_hyper_ball() have been
+  slightly changed. It concerns the points along the straight faces and in the
+  interior. The new positions have been determined to balance the sizes and
+  aspect ratios (expressed by the minimal and maximal singular values of the
+  Jacobian from reference to real coordinates), producting a slightly better
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:956]:
```suggestion
  Jacobian from reference to real coordinates), producing a slightly better
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 8]
```diff
@@ -0,0 +1,2579 @@
...
+  VectorSpaceVector.
+  <br>
+  (Niklas Fehn, 2020/03/06)
+ </li>
+
+ <li>
+  New: Introduce a new communication-pattern class Utilities::MPI::NoncontiguousPartitioner.
+  It is similar to Utilities::MPI::Partitioner, however, does not make any
+  restrictions regarding to the ordering of the underlying index sets. This class enables
+  efficient repartitioning of vectors, which might be benefitial in interfacing with
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:1114]:
```suggestion
  efficient repartitioning of vectors, which might be beneficial in interfacing with
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 9]
```diff
@@ -0,0 +1,2579 @@
...
+ <li>
+  Improved: ParameterHandler::parse_input_from_json() can handle values of type `"a" : 1', in addition to
+  `"a" : {"value" : 1}'.
+  <br>
+  (Peter Munch, 2020/01/27)
+ </li>
+
+ <li>
+  New: ParameterHandler::parse_input_from_json() and  ParameterHandler::parse_input_from_xml() can skip
+  undifined sections and entries.
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:1250]:
```suggestion
  undefined sections and entries.
```

File: doc/news/9.1.1-vs-9.2.0.h [hunk 10]
```diff
@@ -0,0 +1,2579 @@
...
+  unknowns, and one after the cell operation last touches them. The goal of
+  these functions is to bring vector operations close to the time when the
+  vector entries are accessed by the cell operation which increases the cache
+  hit rate of modern processors (temporal locality).
+  <br>
+  (Martin Kronbichler, 2019/09/10)
+ </li>
+
+ <li>
+  New: compare_and_apply_mask() allows to use a ternary operater idiom with
```
Comment (drwells) [inline doc/news/9.1.1-vs-9.2.0.h:1965]:
```suggestion
  New: compare_and_apply_mask() allows to use a ternary operator idiom with
```

**Comment by drwells** [comment]
Well, those fixes managed to give Travis a heart attack. My apologies, everyone, for crashing the train with some minor fixes. Let me put up a separate PR for this content.

**Comment by drwells** [comment]
Travis is stuck. Here is the output:
```
[100%] Generating documentation via doxygen.

-- Documentation is available at /home/travis/build/dealii/dealii/build/doc/doxygen/deal.II/index.html

[100%] Built target doxygen

Scanning dependencies of target documentation

[100%] Built target documentation

The command "make -j documentation" exited with 0.

0.00s$ cat doxygen.log && ! [ -s doxygen.log ]

The command "cat doxygen.log && ! [ -s doxygen.log ]" exited with 0.
cache.2

store build cache

Done. Your build exited with 0.
````

**Pull request merged**